build/constants.yaml

```diff
@@ -4,6 +4,7 @@
     application-insights-instrumentation-key-env-var-name: ORYX_AI_INSTRUMENTATION_KEY
     app-service-app-name-env-var-name: APPSETTING_WEBSITE_SITE_NAME
     scm-commit-id-env-var-name: SCM_COMMIT_ID
+    php-origin-env-var-name: PHP_ORIGIN
```

Comment:
add docs - what is it used for? who defines it? who consumes it?

Reply:
sure ..